src/openzaak/components/documenten/tests/admin/test_eio.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from openzaak.utils.tests import AdminTestMixin
+
+
+class EnkelvoudigInformatieObjectAdminTests(AdminTestMixin, TestCase):
+    heeft_alle_autorisaties = True
```

Comment:
this attribute is normally only used for API tests, no?

Reply:
whoops, true, bit too much copypasta